server/views/pages/createGoals/index.njk

```diff
@@ -171,15 +172,17 @@
                   </strong>
                 </div>
                 {{ errors | findError('note') }}
-                <textarea class="govuk-textarea" id="note" name="note" rows="5" aria-describedby="warning-text">{{ form.note }}</textarea>
+                <textarea class="govuk-textarea" id="note" name="note" rows="5" aria-describedby="warning-text">{{ form.goals[0].note }}</textarea>
```

Comment:
As above, the `name`, `id` and `findErrors` will need to be something like `goal[0].note`